install/kubernetes/helm/subcharts/gateways/values.yaml

```diff
@@ -6,6 +6,7 @@
 # Disable specifc gateway by setting the `enabled` to false.
 #
 enabled: true
+nodeSelector: {}
```

Comment:
do we really want to let user configure nodeSelector here vs within each of the gateway?  I'd vote for the later to be consistent as the rest of attributes plus user may want to do different nodeSelector for multiple gateways.

Reply:
Updated `nodeSelector` for gateway, we can now add `nodeSelector` for each gateway independently.